mitiq/ddd/tests/test_insertion.py

```diff
@@ -253,3 +253,17 @@ def test_get_slack_matrix_from_circuit__bad_input_errors():
 def test_insert_sequences(circuit, result, rule):
     circuit_with_sequences = insert_ddd_sequences(circuit, rule)
     assert circuit_with_sequences == result
+
+
+def test_midcircuit_measurement_raises_error():
+    qreg = qiskit.QuantumRegister(2)
```

Comment:
Do we need to test this on another frontend besides Qiskit?

Reply:
Hmmm good question! I'm not actually too sure. I think since the code we are ultimately trying to test is written for cirq, and we have a test that we know covers it (even though it's in qiskit), it _should_ be fine, but what do you think?

Reply:
Agree Cirq doesn't need an explicit test. For the other supported frontends, I think it should be sufficient to check the behavior with other frontends "offline" and record it in a comment on this PR.

Reply:
Added a pyquil test in [`55fae24`](https://github.com/unitaryfund/mitiq/pull/1586/commits/55fae247c747916b01bf3758de4cb928e32dffc1). Looks like neither pennylane, nor braket support mid circuit measurement currently.

Reply:
That works!